pkg/ddc/jindofsx/transform.go

```diff
@@ -211,22 +211,33 @@ func (e *JindoFSxEngine) transformMaster(runtime *datav1alpha1.JindoRuntime, met
 	propertiesFileStore := map[string]string{}
 
 	for _, mount := range dataset.Spec.Mounts {
-		if !strings.HasSuffix(mount.MountPoint, "/") {
-			mount.MountPoint = mount.MountPoint + "/"
-		}
 		// support nas storage
-		if strings.HasPrefix(mount.MountPoint, "local:///") {
-			value.Mounts.Master[mount.Name] = &Level{
-				Path: mount.MountPoint[8:],
-				Type: "hostPath",
+		if strings.HasPrefix(mount.MountPoint, "pvc://") {
```

Comment:
I think it's better to use `common.VolumeScheme` instead of use constant `pvc://`

Reply:
fuse also need to access to pvc as client